### DISCUSSION TITLE
Simu5G Cars Error

### DISCUSSION THREAD

**Discussion by eykaamsha** [omnetpp/omnetpp#1265] [General]
Hello, I'm trying to run the cars example in the `simu5g/simulation/nr/cars` directory using OMNeT++ 5.6.2, INET 4.2.2, Simu5G 1.1.0, and Veins 5.2. However, after running the simulation, the car nodes do not appear in the network. Has anyone encountered this issue or knows how to resolve it?

**Comment by Muhammaduazir69** [comment]
Enable simu5g projects feature 'Cars'

**Reply by eykaamsha** [reply]
I’ve enabled it but still cars do not appear
